File: 392. Is Subsequence/Solution.py
class Solution:
    def isSubsequence(self, s: str, t: str) -> bool:
      j = 0
      lst =""
      for i in range(len(s)):
        flag = True
        while flag:
          if j >= len(t):
            return False
          if s[i] == t[j]:
            print(j)
            lst = lst + s[i]
            flag = False
            j += 1
          else:
            j += 1
      print(lst)
      if lst == s:
        return True

File: 392. Is Subsequence/Solution.c
#include <string.h>
bool isSubsequence(char * s, char * t){
  int j = 0;
  for(int i =0 ; i<strlen(t) ; i++){
    if(j<strlen(s) && s[j] == t[i]){
      j += 1;
    }
  }
  return j == strlen(s);
}